python/tank/deploy/tank_commands/move_pc.py

```diff
@@ -97,7 +97,12 @@ def _copy_folder(self, log, level, src, dst):
     
     
     def run_interactive(self, log, args):
-        
+
+        if self.tk.pipeline_configuration.is_unmanaged():
```

Comment:
I like the idea, but a lot of the commands that we are making changes are now are effectively becoming legacy commands that are in some way replaced by bootstrap equivalents - setup, localize, share core, push, clone, move... all are now part of a new framework. We have some tickets to go over the tank commands and make sure they are all making sense and using the new code - I think this is a good opportunity to dive a bit deeper here too and maybe discuss this. For now, i just added the obvious little catches..